### PR TITLE
Deprecate IBMRuntimeService, IBMEstimator, IBMSampler classes

### LINKED ISSUE #260: Rename IBMRuntimeService/IBMEstimator/IBMSampler classes
**What is the expected feature or enhancement?**
1. Rename IBMRuntimeService -> QiskitRuntimeService (Deprecate IBMRuntimeService class name)
2. Deprecate IBMEstimator/IBMSampler classes and have the users directly use the current Estimator/Sampler classes. (with only change of backend will now be passed under options parameter)

Old:
```python
from qiskit_ibm_runtime import IBMRuntimeService
service = IBMRuntimeService(channel="ibm_cloud", token="abc", instance="crn")

estimator_factory = IBMEstimator(service=service, backend="ibm_algiers")

with estimator_factory(circuits=[qc], observables="...", parameters="...") as estimator:
    # estimator here is instance of Estimator class
```

New:
```python
from qiskit_ibm_runtime import QiskitRuntimeService
service = QiskitRuntimeService(channel="ibm_cloud", token="abc", instance="crn")

with Estimator(circuits=[qc], observables="...", parameters="...", options={ "backend": str | Backend }, service=service) as estimator:
    result = estimator(circuit_indices=[0], ...)
```

**Acceptance criteria**

### PR DESCRIPTION
**Pull request by rathishcholarajan**
### Summary
See [issue](https://github.com/Qiskit/qiskit-ibm-runtime/issues/260) description.

### Details and comments
Fixes #260

### CODE REVIEW

File: qiskit_ibm_runtime/estimator.py [hunk 1]
```diff
@@ -19,21 +19,117 @@
...
+    Qiskit Runtime Estimator primitive service estimates expectation values of quantum circuits and
+    observables.
+
+    Estimator can be initialized with the following parameters.
+
+    * circuits: a (parameterized) :class:`~qiskit.circuit.QuantumCircuit` or
+        a list of (parameterized) :class:`~qiskit.circuit.QuantumCircuit`.
+
+    * observables: a list of :class:`~qiskit.quantum_info.SparsePauliOp`.
+
```
Comment (daka1510) [inline qiskit_ibm_runtime/estimator.py:40]: The source of the changes in this file (and the relation to the scope of the PR) is not clear to me. Is that copied from [qiskit-terra](https://github.com/Qiskit/qiskit-terra/tree/main/qiskit/primitives)

File: test/integration/test_estimator.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Integration tests for Estimator primitive."""
```
Comment (daka1510) [inline test/integration/test_estimator.py:13]: I'm not sure about the difference between `test_estimator` and `test_ibm_estimator` (likely due to lack of context on my side). Why do we need both?

File: qiskit_ibm_runtime/estimator.py (hunk 1 above)
Comment (rathishcholarajan) [reply qiskit_ibm_runtime/estimator.py:40]: Estimator becomes the primary interface for users instead of IBMEstimator. Issue description has more details.

Comment (rathishcholarajan) [reply qiskit_ibm_runtime/estimator.py:40]: This is copied from IBMEstimator class.

**Pull request merged**